composer.json

```diff
@@ -1,5 +1,20 @@
 {
     "require": {
-        "google/apiclient": "^2.5"
+        "php": ">=7.2.5",
+        "google/apiclient": "^2.5",
+        "google/apiclient-services": "v0.138"
+    },
+    "replace": {
+        "monolog/monolog": "*"
```

Comment:
This is to prevent installing monolog, as it's already available in core. That was removed manually before.